### PR TITLE
Add verify CLI command

### PR DESCRIPTION
**Pull request by teake**
This is useful for checking whether all the tests in the
test report(s) were either successful or were skipped.

### CODE REVIEW

File: junitparser/cli.py [hunk 1]
```diff
@@ -16,6 +16,20 @@ def merge(paths, output):
     return 0
 
 
+def verify(paths):
+    """Verify is none of the testcases failed or errored."""
```
Comment (EnricoMi) [inline junitparser/cli.py:20]:
```suggestion
    """Verify if none of the testcases failed or errored."""
```

File: junitparser/cli.py [hunk 2]
```diff
@@ -16,6 +16,20 @@ def merge(paths, output):
...
+    # We could grab the number of failures and errors from the statistics of the root element
+    # or from the test suites elements, but those attributes are not guaranteed to be present
+    # not guaranteed to be correct. So we'll just loop over all the testcases.
```
Comment (EnricoMi) [inline junitparser/cli.py:23]:
```suggestion
    # We could grab the number of failures and errors from the statistics of the root element
    # or from the test suites elements, but those attributes are not guaranteed to be present
    # or correct. So we'll just loop over all the testcases.
```

File: junitparser/cli.py [hunk 3]
```diff
@@ -16,6 +16,20 @@ def merge(paths, output):
     return 0
 
 
+def verify(paths):
```
Comment (EnricoMi) [inline junitparser/cli.py:19]: would a test calling this method be useful?

**Review by EnricoMi — COMMENTED** [review]
minor comments

File: junitparser/cli.py (hunk 1 above)
Comment (teake) [reply junitparser/cli.py]: Thanks!

File: junitparser/cli.py (hunk 2 above)
Comment (teake) [reply junitparser/cli.py]: Thanks again!

File: junitparser/cli.py (hunk 3 above)
Comment (teake) [reply junitparser/cli.py:19]: Yes, it would indeed. I've added the test.

**Review by weiwei — APPROVED** [review]
LGTM. Thanks for the PR.

**Review by weiwei — APPROVED** [review]
LGTM. Thanks for the PR.

**Pull request merged**